Products/CMFPlone/browser/contact_info.py

```diff
@@ -67,12 +67,13 @@ def send_message(self, data):
 
         try:
             # This actually sends out the mail
-            host.send(
+            host.secureSend(
```

Comment:
Why the change to `secureSend`? This method  from SecureMailHost was already deprecated in Plone 4.  It is still there, but we want to get rid of it. See issue #965.